src/Geometry.cpp.Rt

```diff
@@ -492,8 +492,8 @@ inline int Geometry::loadSTL(lbRegion reg, pugi::xml_node n)
     ret = fread(header, 80, sizeof(char), f);
     ret = fread(&ntri, 1, sizeof(int), f);
     tri = (STL_tri *) malloc(ntri * sizeof(STL_tri));
-    char *lev = (char *) malloc(reg.size() * sizeof(char));
-    for (int i = 0; i < reg.size(); i++)
+    char *lev = (char *) malloc(reg.sizeL() * sizeof(char));
+    for (size_t i = 0; i < reg.size(); i++)
```

Comment:
`size()` have to be changed to `sizeL()` to match the type of `i`. 

`size()` returns an `int`, and `sizeL()` returns a `size_t`